example_usage_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/simplesurance/proteus/sources/cfgenv"
 )
 
-func ExampleParsed_Usage() {
+func ExampleParsed_Help() {
```

Comment:
Not sure what to do with the example function, should it only call usage or help?
Or, like currently, show how to print usage+help